mobile/src/bundles/common/components/radio-buttons/radio-buttons.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import {
+    type Control,
+    type FieldPath,
+    type FieldValues,
+} from 'react-hook-form';
+import RadioGroup, {
+    type RadioButtonProps,
+    type RadioGroupProps,
+} from 'react-native-radio-buttons-group';
+
+import { View } from '~/bundles/common/components/components';
+import { Color } from '~/bundles/common/enums/enums';
+import { useFormController } from '~/bundles/common/hooks/hooks';
+
+type Properties<T extends FieldValues> = RadioGroupProps & {
+    radioButtons: RadioButtonProps[];
+    name: FieldPath<T>;
+    control: Control<T, null>;
+};
```

Comment:
```suggestion
type Properties<T extends FieldValues> = RadioGroupProps & {
    name: FieldPath<T>;
    control: Control<T, null>;
};
```

RadioGroupProps already has radioButtons props inside